TYPO3.Flow/Classes/TYPO3/Flow/Core/Bootstrap.php

```diff
@@ -526,6 +527,12 @@ protected function defineConstants()
         define('FLOW_PATH_DATA', FLOW_PATH_ROOT . 'Data/');
         define('FLOW_PATH_PACKAGES', FLOW_PATH_ROOT . 'Packages/');
 
+        if (!defined('FLOW_PATH_TEMPORARY')) {
+            define('FLOW_PATH_TEMPORARY_BASE', self::getEnvironmentConfigurationSetting('FLOW_PATH_TEMPORARY_BASE') ?: FLOW_PATH_DATA . '/Temporary');
+            $temporaryDirectoryPath = Files::concatenatePaths(array(FLOW_PATH_TEMPORARY_BASE, str_replace('/', '/SubContext', (string)$this->context))) . '/';
+            define('FLOW_PATH_TEMPORARY', $temporaryDirectoryPath);
```

Comment:
This is now simplified as follows:
* We only evaluate the `FLOW_PATH_TEMPORARY_BASE` env variable (as documented)
* Accordingly only the `FLOW_PATH_TEMPORARY_BASE` is now passed to the sub requests (therefore we need to set the `FLOW_PATH_TEMPORARY_BASE` *constant*, too)
* The `FLOW_PATH_TEMPORARY` constant is set for convenience, so that it can be used instead of having to inject `Environment`